spec/lib/miq_automation_engine/miq_ae_discovery_spec.rb

```diff
@@ -24,10 +26,14 @@ module MiqAeDiscoverySpec
       let(:options) { {'test' => true} }
 
       it "check automate parameters" do
-        attrs = {:event_id          => @event.id,
-                 :event_type        => @event.event_type,
-                 "VmOrTemplate::vm" => @vm.id,
-                 :vm_id             => @vm.id}
+        attrs = {:event_id                  => @event.id,
+                 :event_type                => @event.event_type,
+                 "ExtManagementSystem::ems" => @ems.id,
+                 :ems_id                    => @ems.id, 
+                 "VmOrTemplate::vm"         => @vm.id,
+                 :vm_id                     => @vm.id,
+                 :tenant_id                 => @tenant.id,
```

Comment:
@mkanoor are we still adding tenant_id here?
@lfu is there any chance we can get a `user` or a `group`?

Maybe `@vm.miq_group`?